Plugins/Flow.Launcher.Plugin.Explorer/Views/ActionKeywordSetting.xaml.cs

```diff
@@ -19,22 +19,22 @@ public partial class ActionKeywordSetting : Window
 
         public string ActionKeyword
         {
-            get => _actionKeyword;
+            get => actionKeyword;
             set
             {
                 // Set Enable to be true if user change ActionKeyword
-                if (Enabled is not null)
-                    Enabled = true;
-                _actionKeyword = value;
+                Enabled = true;
+                actionKeyword = value;
             }
         }
 
-        public bool? Enabled { get; set; }
+        public bool Enabled { get; set; }
 
-        private string _actionKeyword;
+        private string actionKeyword;
 
-        public Visibility Visible =>
```

Comment:
Hmmm I think why not keep it simple. Adding an option to disable FileContentSearch actionkeyword won't be a problem to user.